spec/requests/get_one_institution_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Institutions', type: :request do
+  describe 'GET /institutions/get_one_institution' do
+    before do
+      type = create(:institution_type)
+      (1..60).each do |_n|
+        create(:institution, institution_type: type)
+      end
```

Comment:
You can use create_list instead of an explicit loop
```suggestion
      create_list(:institution, 60, institution_type: type)
```
Do you need 60 records in each example here, though? This approach will eventually make tests pretty slow and annoying